.github/workflows/docker-build.yaml

```diff
@@ -13,7 +13,13 @@ on:
         required: false
         type: string
         default: .
+      file:
+        description: "name of Dockerfile"
+        required: false
+        type: string
+        default: Dockerfile
       generate-token:
+        description: "generates token"
```

Comment:
Mayve write: Generate token if its enabled. This token is used in case f.e. docker build need access to github repos like case with composer.